### DISCUSSION TITLE
Does anyone make it work with NVIDIA RTX 4090?

### DISCUSSION THREAD

**Discussion by venturaEffect** [suno-ai/bark#458] [General]
I've wrote already more than 10 days ago that it doesn't work with a NVIDIA RTX 4090. It is a bit frustrating having the last GPU and not be able to use Bark

Does anyone know how to make it work?

BTW, appreciate the product you've built, just want to be able to use it.

Thanks

**Comment by fabhatti** [comment]
Mine is working fine with Zoltrix 4090 card.

**Reply by fabhatti** [reply]
If I recall, when I installed it, I got a prompt about GPU usage. Did you get one when you installed yours?

**Comment by donaldafeith** [comment]
If you haven't already check the wheels you're installing. Make sure you're installing with Cuda support. You probably have already done this. It would be helpful to know what OS you're using.
